apps/src/templates/progress/ProgressPill.story.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import ProgressPill from './ProgressPill';
+import { LevelStatus } from '@cdo/apps/util/sharedConstants';
+
+export default storybook => {
+  storybook
+    .storiesOf('ProgressPill', module)
+    .addStoryTable([
```

Comment:
True. Maybe just a matter of exporting this stories, and then importing them in a unit test? I'm not going to do that in this PR, but may explore that in a separate future PR.